gitian-building/gitian-building-setup-gitian-fedora.md

```diff
@@ -80,23 +80,4 @@ git clone https://github.com/bitcoin-core/gitian.sigs.git
 git clone https://github.com/bitcoin-core/bitcoin-detached-sigs.git
 ```
 
-Setting up the Gitian image
--------------------------
-
-Gitian needs a virtual image of the operating system to build in.
-Currently this is Ubuntu Trusty x86_64.
-This image will be copied and used every time that a build is started to
-make sure that the build is deterministic.
-Creating the image will take a while, but only has to be done once.
-
-Execute the following as user `gitianuser`:
-
-```bash
-cd gitian-builder
-git checkout 686a00ad712e30ba3a7850c16ef32f650df5b5dc # This version seems to work better than master
-bin/make-base-vm --lxc --arch amd64 --suite trusty
```

Comment:
Why is this removed? The build script can only create a bionic image and when running without the build script, this step is required.

Reply:
If you do this step here, the build script fails because the image is already present.

If you run without the build script, you find the step in gitian-building-manual.md.

Reply:
BTW, my aim was this: https://github.com/bitcoin/bitcoin/issues/12511#issuecomment-385471954

Reply:
Ah, I see. Thx.

Reply:
I understand just now...
Leaving the manual step to make-base-vm trusty is a trick to have the gitian setup for older tags.
Instead the gitian-build.py is in charge of executing make-base-vm bionic for 0.17.x and later.
:)
I will fix it the sooner the better.